client/src/config.ts

```diff
@@ -1,21 +1,25 @@
 import getConfig from 'next/config'
 
-const { publicRuntimeConfig } = getConfig()
+const { publicRuntimeConfig} = getConfig()
 //
 // NEVERMINED REMOTE CONNECTIONS
 //
 export const marketplaceUri = process.env.NEXT_PUBLIC_MARKETPLACE_API || 'http://localhost:3100' // next redirections on next.config.js
+export const gatewayURL = process.env.NEXT_PUBLIC_GATEWAY_URI || "http://localhost:8030"
+
 export const gatewayUri = '/api/gateway' // next redirections on next.config.js
 export const faucetUri = '/api/faucet' // next redirections on next.config.js
-export const gatewayAddress =
-  process.env.NEXT_PUBLIC_GATEWAY_ADDRESS || '0x068ed00cf0441e4829d9784fcbe7b9e26d4bd8d0'
-export const nodeUri =
-  process.env.NEXT_PUBLIC_NODE_URI ||
-  'https://polygon-mumbai.g.alchemy.com/v2/GHT7ODcuJCgTuQGyvjI92G6hI5hyJLEa'
+export const gatewayAddress =   process.env.NEXT_PUBLIC_GATEWAY_ADDRESS || '0x068ed00cf0441e4829d9784fcbe7b9e26d4bd8d0'
+export const nodeUri = process.env.NEXT_PUBLIC_NODE_URI || 'http://localhost:8545'
 export const secretStoreUri = process.env.NEXT_PUBLIC_SECRET_STORE_URI || 'http://localhost:12001'
 export const artifactsFolder =
   process.env.NEXT_PUBLIC_ARTIFACTS_FOLDER_URL || `http://localhost:3000/artifacts`
 
+export const tier1NftContractAddress = '0x18bdFAf7Cc2B66a4Cfa7e069693CD1a9B639A69b'
```

Comment:
Is needed a env var for each tier contract

Reply:
Done